src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx

```diff
@@ -35,7 +37,16 @@ export default function ArrivalByTimeChart({
     gtfs_route_date: string
     gtfs_route_hour: string
   }[]
+  operatorId: string
 }) {
+
+  const filteredData = useMemo(
+    () => data.filter((item) => operatorId && item.id === operatorId),
```

Comment:
why not
```suggestion
    () => data.filter((item) => !operatorId || item.id === operatorId),
```
?

Reply:
Damn that's a smart one, haven't thought of that.
Can I still update the pr?

Reply:
sure

Reply:
in fact, you'll need to update it anyway as the linter is failing